dependency-check-maven-config.xml

```diff
@@ -182,4 +182,14 @@
         <cve>CVE-2016-2108</cve>
         <cve>CVE-2016-2109</cve>
     </suppress>
+
+    <!-- Remove this suppression when DDF-3440 is worked -->
```

Comment:
✏️ I would just ask that you include a link back to this line in the Jira ticket so it doesn't get lost.

Reply:
Added notes to the Solr Upgrade tickets to remove the suppression when complete.